### DISCUSSION TITLE
Inquiry Regarding Version 1.1 of Hexagon-Chart

### DISCUSSION THREAD

**Discussion by jkp9339** [nkia-development/hexagon-chart#10] [Q&A] [answered]
Dear Hexagon-Chart Development Team,

I hope this message finds you well. My name is [Your Name], and I am currently utilizing the Hexagon-Chart library for my projects. Firstly, I would like to express my gratitude for your efforts in developing and maintaining this open-source tool.

However, I have noticed that the Hexagon-Chart library is currently marked as version 1.1. Despite this, it seems that some functionalities and features are still incomplete or not fully operational. This has led to some confusion regarding the versioning and the overall readiness of the library for production use.

Could you kindly provide some insights on the following points?

1.	Current Status: What is the current development status of the Hexagon-Chart library? Are there any known incomplete features or functionalities that users should be aware of?
2.	Versioning Rationale: What was the rationale behind designating the current version as 1.1? Typically, a 1.x version implies a certain level of stability and completeness. Understanding your versioning strategy would help users like myself better gauge the maturity of the library.
3.	Roadmap and Updates: Is there a publicly accessible roadmap or timeline for upcoming updates and feature completions? This information would be immensely helpful for planning purposes and for aligning our project timelines with the expected improvements in the library.
4.	Contributions and Feedback: How can the community contribute to the development and improvement of the Hexagon-Chart library? Are there specific areas where contributions are most needed or welcomed?

Thank you for your time and consideration. I look forward to your response and to the continued evolution of the Hexagon-Chart library.

Best regards,

> jkpjkp9339

**Comment by jukangpark** [comment, marked as answer]
Dear jkp9339,

Thank you for reaching out and for your interest in the Hexagon-Chart library.

We appreciate your feedback and would like to address your concerns:

1.	Current Status: The Hexagon-Chart library is still under active development. We acknowledge that some functionalities and features are incomplete or not fully operational at this stage.
2.	Versioning Rationale: The designation of the current version as 1.1 was indeed a mistake. Initially, when we published the library, we inadvertently marked the version as 1.0.0. We apologize for any confusion this has caused.
3.	Roadmap and Updates: We are committed to improving the library and providing a more accurate versioning strategy moving forward. We will ensure that future versions better reflect the stability and completeness of the library. Additionally, we will work on providing a publicly accessible roadmap to keep users informed about upcoming updates and feature completions.
4.	Contributions and Feedback: We welcome and encourage community contributions. Specific areas where contributions are most needed include completing incomplete features, improving documentation, and identifying and fixing bugs. Any feedback or suggestions are also highly appreciated.

Thank you once again for bringing this to our attention. We are dedicated to improving the Hexagon-Chart library and making it a more robust tool for all users.

Best regards,

The Hexagon-Chart Development Team

**Reply by jkp9339** [reply]
Dear Hexagon-Chart Development Team,

Thank you for your prompt and detailed response. I appreciate the clarification regarding the current development status and the initial versioning mistake. It’s reassuring to know that these issues are being addressed.

I am looking forward to the improved versioning strategy and the publicly accessible roadmap. This will greatly help in planning and aligning our project timelines with the expected updates.

Additionally, I am keen to contribute to the project. Could you provide more details on how I can get started with contributing, such as any guidelines or specific areas that need immediate attention?

Thank you once again for your dedication and efforts in developing the Hexagon-Chart library.

Best regards, 

> jkp9339

**Comment by jukangpark** [comment]
Thanks
